code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/filter/ServiceCreateValidationFilter.java

```diff
@@ -49,6 +60,30 @@ public Object create(String type, ApiRequest request, ResourceManager next) {
                     "name");
         }
 
+        // 2. if a part of sidekick, scale should match other sidekick participants scale
+        Map<String, Object> launchConfig = DataAccessor
```

Comment:
Thats actually the thing I wanted to talk to you about. if we don't keep the scale same, its hard to validate what is the desired value (because what if the user scaled down)? and I cant update the scale inside of the process - I ran into way too many race conditions especially with idempotent checks on. Taking the max value would def make things easier.